cli/src/test/java/com/devonfw/tools/ide/commandlet/EnvironmentCommandletTest.java

```diff
@@ -8,33 +8,119 @@
+        IdeLogEntry.ofInfo("IDE_HOME=\"" + context.getIdeHome() + "\""), //
+        IdeLogEntry.ofInfo("export M2_REPO=\"" + context.getUserHome() + "/.m2/repository\""), //
+        new IdeLogEntry(IdeLogLevel.INFO, "export PATH=", true), //
+        IdeLogEntry.ofInfo("WORKSPACE=\"foo-test\""), //
+        IdeLogEntry.ofInfo("WORKSPACE_PATH=\"" + context.getWorkspacePath() + "\"") //
+    );
+  }
+
+  /**
+   * Test of {@link EnvironmentCommandlet} run with INFO logging (plain variables without source).
+   */
+  @Test
+  public void testRunInfoLogging() {
+
+    // arrange
+    String path = "project/workspaces/foo-test/my-git-repo";
+    IdeTestContext context = newContext(PROJECT_BASIC, path, false, IdeLogLevel.INFO);
+    EnvironmentCommandlet env = context.getCommandletManager().getCommandlet(EnvironmentCommandlet.class);
+    Path userProperties = context.getUserHomeIde().resolve("ide.properties");
+    Path settingsIdeProperties = context.getSettingsPath().resolve("ide.properties");
+    Path confIdeProperties = context.getConfPath().resolve("ide.properties");
+    Path workspaceIdeProperties = context.getWorkspacePath().resolve("ide.properties");
+    // act
+    env.run();
+    // assert
+    assertThat(context).log().hasEntriesWithNothingElseInBetween( //
+        IdeLogEntry.ofInfo("BAR=\"bar-some-${UNDEFINED}\""), //
+        IdeLogEntry.ofInfo("DOCKER_EDITION=\"docker\""), //
+        IdeLogEntry.ofInfo("ECLIPSE_VERSION=\"2023-03\""), //
+        IdeLogEntry.ofInfo("FOO=\"foo-bar-some-${UNDEFINED}\""), //
+        IdeLogEntry.ofInfo("HOME=\"" + context.getUserHome() + "\""), //
+        IdeLogEntry.ofInfo("IDE_HOME=\"" + context.getIdeHome() + "\""), //
+        IdeLogEntry.ofInfo("IDE_TOOLS=\"mvn,eclipse\""), //
+        IdeLogEntry.ofInfo("INTELLIJ_EDITION=\"ultimate\""), //
+        IdeLogEntry.ofInfo("JAVA_VERSION=\"17*\""), //
+        IdeLogEntry.ofInfo("export M2_REPO=\"" + context.getUserHome() + "/.m2/repository\""), //
+        IdeLogEntry.ofInfo("MVN_VERSION=\"3.9.1\""), //
+        new IdeLogEntry(IdeLogLevel.INFO, "export PATH=", true), //
+        IdeLogEntry.ofInfo("SOME=\"some-${UNDEFINED}\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS1=\" user1 settings1 workspace1 conf1\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS10=\"user10 workspace10\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS2=\" user2 conf2\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS3=\" user3 workspace3\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS4=\" settings4\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS5=\" settings5 conf5\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS6=\" settings6 workspace6 conf6\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS7=\"user7 settings7 workspace7 conf7\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS8=\"settings8 workspace8 conf8\""), //
+        IdeLogEntry.ofInfo("TEST_ARGS9=\"settings9 workspace9\""), //
+        IdeLogEntry.ofInfo("TEST_ARGSa=\" user1 settings1 workspace1 conf1  user3 workspace3 confa\""), //
```

Comment:
Why is the spacing so odd in the test arguments?

Reply:
What exactly do you mean? IntelliJ formatter? Or do you mean the values of these `TEST_ARGS*` variables?
For the first IMHO all is working as expected and for the latter you need to ask @MattesMrzik who created these variables and their values. IMHO he tried to cover many edge-cases of variable resolution and therefore also added leading or duplicated whitespace, but I am just guessing...

Reply:
user home:
```
TEST_ARGS1=${TEST_ARGS1} user1
TEST_ARGS3=${TEST_ARGS3} user3
```
settings:
```
TEST_ARGS1=${TEST_ARGS1} settings1
```
workspace:
```
TEST_ARGS1=${TEST_ARGS1} workspace1
TEST_ARGS3=${TEST_ARGS3} workspace3
```
conf:
```
TEST_ARGS1=${TEST_ARGS1} conf1
TEST_ARGSa=${TEST_ARGS1} ${TEST_ARGS3} confa
```

So the expected result is correct but rather hard to understand...